src/main/java/no/nav/veilarboppfolging/client/veilarbarena/VeilarbarenaClientImpl.java

```diff
@@ -22,24 +25,29 @@
 @Slf4j
 public class VeilarbarenaClientImpl implements VeilarbarenaClient {
 
+    private static final DownstreamApi veilarbArenaApi = new DownstreamApi(EnvironmentUtils.requireClusterName(), "pto", "veilarbarena");
```

Comment:
Name `DownstreamApi` is bit misleading since I would expect some api, and its only data class...